ncmcm/bundlenet/bundlenet.py

```diff
@@ -34,29 +47,38 @@ class BunDLeNet(Model):
             discrete behavioural states
             For continuous-valued behaviours, this stipulates the number of
             behaviour variables
+
+        input_shape (tuple):
```

Comment:
could you also add the other arguments as well to the docstring?

Reply:
_latent_dim_ and _num_behaviour_ are actually already there, they're just not shown in the PR diff